src/styles/sass/partials/_typography.scss

```diff
@@ -0,0 +1,110 @@
+@import url("https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&family=Fasthand&display=swap");
+
+h1 {
+	@include setTypography($ff-primary, $fw-regular, $fs-800);
+	color: $clr-white;
+	background-color: $clr-black;
+	text-shadow: (4px 4px 4px $clr-dark-blue);
+	padding: 10px 0;
+	margin: 30px 0;
+}
+
+// h2 - my toolkit, recent projects, my journey
+h2 {
+	@include setTypography($ff-secondary, $fw-regular, $fs-700);
+	text-transform: uppercase;
+	color: $clr-white;
+	text-shadow: (1px 1px 4px $clr-pink);
+	padding: 10px;
+	text-align: center;
+	margin: 20px 0;
+	background-color: $clr-black;
+	width: 100%;
+}
+
+// h3 - I'm a front end, who am i here are some, Hacker News, please feel free
```

Comment:
Here's another comment from your previous project

Reply:
I borrowed my typography partial from my portfolio and completely forgot to update the comments. Thanks for catching this.